### PR TITLE
Inventory Click event-value null check

### PR DESCRIPTION
**Pull request by Asleeepp**
### Description

adds a null check to the event-slot of the inventory click event
I tested it, it just returns air when clicking outside the inventory, which should be fine (probably)

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** https://github.com/SkriptLang/Skript/issues/6458

### CODE REVIEW

**Review by Moderocky — APPROVED** [review]

**Comment by Moderocky** [comment]
Ah nuts I switched it to the wrong branch, give me a few minutes to fix the mess 😬

**Comment by Asleeepp** [comment]
> Ah nuts I switched it to the wrong branch, give me a few minutes to fix the mess 😬

i was so confused when it suddenly switched to 23 commits

**Comment by Moderocky** [comment]
Okay *now* I shall try to press the right button 🤣

File: src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java [hunk 1]
```diff
@@ -1123,6 +1123,9 @@ public ItemStack get(final InventoryClickEvent e) {
 			@Nullable
 			public Slot get(final InventoryClickEvent e) {
 				Inventory invi = e.getClickedInventory(); // getInventory is WRONG and dangerous
+				if (invi == null) {
+					return null;
+				}
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java:1128]:
```suggestion
				if (invi == null)
					return null;
```

**Review by sovdeeth — CHANGES_REQUESTED** [review]
wee woo wee woo brace police

**Review by sovdeeth — APPROVED** [review]

**Comment by Blizzmond** [comment]
where can I download the latest skript.jar that contains this fix?

**Comment by sovdeeth** [comment]
checks tab -> java 17 -> artifacts -> download the zip

**Comment by Blizzmond** [comment]
sorry im not too familiar with github, could you send a link or further guidiance? apologies.

**Comment by sovdeeth** [comment]
checks tab
image
java 17
image
artifacts -> download zip
image

**Pull request merged**